Tests/Contoso.Test.Functional.iOS/AppDelegate.cs

```diff
@@ -19,6 +19,9 @@ public class AppDelegate : RunnerAppDelegate
     {
         private const string ResultChannelHost = "127.0.0.1";
 
+        private UIApplication UiApplication;
+        private NSDictionary LaunchOptions;
```

Comment:
These fields are not used.

Reply:
Even if the field is used, I would name it `UI` instead of `Ui`.